modules/nvidia_plugin/tests/functional/shared_tests_instances/single_layer_tests/convert_color_common.hpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "common_test_utils/ov_tensor_utils.hpp"
+
+namespace ov {
+namespace test {
+namespace nvidia_gpu {
+
+class ConvertColorCUDALayerTest : virtual public ov::test::SubgraphBaseTest {
+private:
+    ov::Tensor generate_input(const ov::element::Type element_type, const ov::Shape& shape) const {
+        return utils::create_and_fill_tensor(element_type, shape, 255);
+    };
+    void generate_inputs(const std::vector<ov::Shape>& target_input_static_shapes) override {
```

Comment:
Do we really should overload `generate_inputs` method? Looks like the code is based on default scenario from `SubgraphBaseTest`

[SubgraphBaseTest -> `generate_inputs` ](https://github.com/openvinotoolkit/openvino/blob/d6c2a10b383dc63563896486906e127f60b5a194/src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp#L240C47-L240C47)
[`generate map`](https://github.com/openvinotoolkit/openvino/blob/d6c2a10b383dc63563896486906e127f60b5a194/src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp#L51)

Reply:
> SubgraphBaseTest

I introduced it to keep test logic:
```
    InferenceEngine::Blob::Ptr GenerateInput(const InferenceEngine::InputInfo& info) const override {
        return FuncTestUtils::createAndFillBlob(info.getTensorDesc(), 255);
    }
```

Reply:
I see, but I ask you because we migrated the input generation logic to [map](https://github.com/openvinotoolkit/openvino/blob/d6c2a10b383dc63563896486906e127f60b5a194/src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp#L51) to generate inputs to any graph based on op types after input.

we have offline discussion with @nkogteva -> @olpipi Could you please finish the PR?

Reply:
Moved to separate ticket 122480